src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -356,7 +309,7 @@ private async Task OnPartialValidationSucceededAsync(ChainedHeaderBlock chainedH
 
                 if (fullValidationRequired)
                 {
-                    connectBlocksResult = await this.FullyValidateLockedAsync(chainedHeaderBlock).ConfigureAwait(false);
+                    connectBlocksResult = await this.TrySwitchTipAfterFullValidationLocked(chainedHeaderBlock).ConfigureAwait(false);
```

Comment:
Methods that start with `try` usually return `bool`. Here it's not the case. 
`Async` suffix is missing. 

I think old name was perfectly fine, no need to change it.